recipes/maoto_agent/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "maoto_agent" %}
+{% set version = "1.0.10b74" %}
+{% set python_min = "3.10" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/maoto_agent-{{ version }}.tar.gz
+  sha256: 05bd9d37b6df3c1b5afd7cb0d0b8646748650ce20293b4a7cfd2d62fa9af8248
+
+build:
```

Comment:
This looks like it should be `noarch: python`. See:
https://conda-forge.org/docs/maintainer/knowledge_base/#noarch-python